drivers/scheduler/scheduler.go

```diff
@@ -424,3 +428,25 @@ func Get(name string) (Driver, error) {
 		Type: "Scheduler",
 	}
 }
+
+// CSISnapshotRequest contains the necessary info to create a CSI snapshot for validation purpose
+type CSISnapshotRequest struct {
+	Timestamp			string
+	Namespace			string
+	OriginalPVCName 	string
+	SnapName 			string
+	RestoredPVCName 	string
+	StorageclassName	string
+	SnapshotclassName	string
+	FADA				bool
```

Comment:
Can we call this `IsFlashArrayDirectAccess`? `FADA` might not make sense to people outside our team.

Also, the field formatting looks a little wonky here, might just be github but would be worth running `go fmt ./drivers/...`